Dockerfiles/build-ebpf-objects.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.18
+# clang and llvm are needed by bpf2go.
+# gcc-multilib is needed for <asm/types.h>.
+# libelf-dev is needed to compile libbpf.
+RUN apt-get update && apt-get install -y clang llvm gcc-multilib libelf-dev
```

Comment:
Can we stick to specific versions of those packages? (at least, clang-11, llvm-11)